src/components/MessagesList/MessagesGroup/MessagesSystemGroup.vue

```diff
@@ -223,6 +230,11 @@ export default {
 					if (isLastReadInsideGroup) {
 						groups.at(-1).collapsed = false
 					}
+
+					// Update collapsed state for the last group if it's not defined yet
+					if (JSON.stringify(message) === JSON.stringify(messages.at(-1)) && groups.at(-1).collapsed === undefined) {
+						groups.at(-1).collapsed = groups.at(-1).messages.length > 1
+					}
```

Comment:
Do we need deep equal check via JSON stringify here? Isn't `message` an object from `messages`?

Reply:
Could you give an example, in which case do we need this `if`?

Reply:
> Do we need deep equal check via JSON stringify here? Isn't message an object from messages?

Yes, for objects comparison ( JS compares by ref) , it will be always faulty here (they are not referencing to the same object) unless you use JSON stringify  which makes them strings (primitive type) to reach the expected comparison. 

> Could you give an example, in which case do we need this if?

It is needed to evaluate the last group as the loop won't go to the first conditional statement in the last message of the last group (given that it has more than 1 message) .  This is if group is not recorded in `groupIsCollapsed` (it was not manually toggled before) , is not `call_reconnected` type and it does not include unread message. It is like a fallback statement so one single message is not marked as collapsed (it can never be collapsed) and it used to be marked collapsed.

One thing to reduce is just putting `groups.at(-1).collapsed = true` instead because condition is passed for the last group that has more than one message.